test/WebJobs.Script.Tests/CSharpEndToEndTests.cs

```diff
@@ -32,6 +32,25 @@ public CSharpEndToEndTests(TestFixture fixture) : base(fixture)
         }
 
         [Fact]
+        public async Task NotificationHub()
+        {
+            await NotificationHubTest();
+        }
+
+        [Fact]
+        public async Task NotificationHub_Out_Notification()
+        {
```

Comment:
I think you can just do:

TouchProjectJson()
NotificationHubTest()

I think the other stuff is just duplicated from your NH test method.